autogen/agentchat/contrib/multimodal_conversable_agent.py

```diff
@@ -85,6 +82,14 @@ def _message_to_dict(message: Union[Dict, List, str]) -> Dict:
             return {"content": message}
         if isinstance(message, dict):
             assert "content" in message, "The message dict must have a `content` field"
+            # GPT-4 does not support function calls yet.
+            message = message.copy()
+            _is_tc = message.pop("tool_calls", None)
+            _is_tr = message.pop("tool_responses", None)
+            if _is_tc or _is_tr:
```

Comment:
Is using private variables in methods a Python convention?

Reply:
Using an underscore `_` before a variable name indicates that it is a temporary or unimportant variable, serving the purpose of readability. They are not considered private member variables in this context because they are not prefixed with `self.`

"Use one leading underscore only for non-public methods and instance variables," as stated in [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names). In this case, it is more of a personal convention.